vulnfeeds/cves/versions.go

```diff
@@ -256,3 +271,30 @@ func CPEs(cve CVEItem) []string {
 	}
 	return cpes
 }
+
+// Parse a well-formed CPE string into a struct.
+func ParseCPE(cpe_str string) (*CPE, bool) {
+	if !strings.HasPrefix(cpe_str, "cpe:") {
+		return nil, false
+	}
+
+	cpe_fields := strings.Split(cpe_str, ":")
```

Comment:
Wait, actually this should be `cpeFields` to conform to Go variable naming conventions.

Reply:
Oh yeah. Thanks.